openasip/newlib-1.17.0/newlib/libc/sys/linux/linuxthreads/machine/i386/useldt.h

```diff
@@ -19,9 +19,8 @@
    write to the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
    Boston, MA 02111-1307, USA.  */
 
-#include <stddef.h>	/* For offsetof.  */
-#include <stdlib.h>	/* For abort().  */
-
+#include <cstddef> /* For offsetof.  */
+#include <cstdlib> /* For abort().  */
```

Comment:
Please avoid touching the newlib files (this is a copy from upstream which we might some day want to synchronize to a newer version so it's best to minimize the diff). Also, this is a C library.

Reply:
I'll revert newlib. (Strange, my changes weren't supposed to touch the C files.)